src/streams/StreamList.js

```diff
@@ -88,6 +88,7 @@ export default function StreamList(props: Props): Node {
           name={item.name}
           iconSize={16}
           isPrivate={item.invite_only}
+          isWebPublic={item.is_web_public}
```

Comment:
(BTW I mentioned this to you on our call yesterday, but just to put it here too: the fact you have to thread this new stream property explicitly through *quite* so many layers is I think a symptom of another misfactoring in some of these components, which is that they should take a Stream and/or Subscription object directly rather than a selection of their properties. That's related to the misfactoring #3767 that I just mentioned, and might be something I pick up and fix soon.)